v2/api/resources/v1beta20200601/resourcegroup_types.go

```diff
@@ -179,7 +179,7 @@ func (status *ResourceGroupStatus) PopulateFromARM(owner genruntime.ArbitraryOwn
 	status.ManagedBy = typedInput.ManagedBy
 	status.Name = typedInput.Name
 	status.Tags = typedInput.Tags
-
+	// Set property ‘AccessTier’:
```

Comment:
Were you getting rid of this?

Reply:
Fixed in my ResourceGroup PR - possibly reverted here by my rebase. Will double check.

Reply:
Was reverted by rebase; will fix.